### PR TITLE
merge_tools.toml: add VS Code as a default merge tool.

### PR DESCRIPTION
**Pull request by ilyagr**
Thanks to @glencbz for noticing that VS Code works fine now as a merge tool, and thanks to @solson for suggesting
`merge-tool-edits-conflict-markers = true`.

# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [x] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json) (doesn't seem necessary)
- [ ] I have added tests to cover my changes

### CODE REVIEW

File: cli/src/config/merge_tools.toml
Comment (martinvonz) [inline cli/src/config/merge_tools.toml:32]: nit: add newline

**Review by martinvonz — APPROVED** [review]
Hopefully someone who has actually tried it can review this, but you have my approval here so you can merge this if you don't get any other reviews.

File: cli/src/config/merge_tools.toml
Comment (ilyagr) [reply cli/src/config/merge_tools.toml]: Done.

**Comment by ilyagr** [comment]
I'll wait in case somebody wants to take a look.

**Review by glencbz — APPROVED** [review]
I haven't pulled this PR and tested, but this looks identical to my current configuration and IMO the experience is exactly what I expect. Thanks!

File: docs/config.toml
Comment (martinvonz) [inline docs/config.toml:18]: Please squash into the parent commit

Comment (ilyagr) [reply docs/config.toml:18]: Yes, I plan to. This will be a bit of a mess for a bit, I tried editing the PR using `github.dev`. Some of the new changes will be in a separate PR.

**Pull request merged**
